testsuite/features/support/env.rb

```diff
@@ -45,6 +45,8 @@
 $custom_download_endpoint = ENV.fetch('CUSTOM_DOWNLOAD_ENDPOINT', nil) if ENV['CUSTOM_DOWNLOAD_ENDPOINT']
 $no_auth_registry = ENV.fetch('NO_AUTH_REGISTRY', nil) if ENV['NO_AUTH_REGISTRY']
 $auth_registry = ENV.fetch('AUTH_REGISTRY', nil) if ENV['AUTH_REGISTRY']
+$current_user = 'admin'
+$current_password = 'admin'
```

Comment:
Will this be changed somewhere? ATM, I do not get how your changes are different to what we already have? I mean, with the changes, we still use the `admin` user. Could you please elaborate this further? Thank you :wink:

Reply:
`current user` and `password` are set in the create user step ( only use during BV smoke test so far ).
https://github.com/uyuni-project/uyuni/blob/a2118e66782e3a880ea7defb5c08792b16bffed7/testsuite/features/step_definitions/api_common.rb#L139-L152
Because I want to keep those steps compatible with CI using admin, I need to setup a default current_user and current_password in this env file that will be overwritten by the create user step when needed.

Reply:
This step does change those global variables and IIRC there are other PRs from Maxime which are going to use it

```ruby
Given(/^I create a user with name "([^"]*)" and password "([^"]*)"/) do |user, password|
  $current_user = user
  $current_password = password
  next if $api_test.user.list_users.to_s.include? user

  $api_test.user.create(user, password, user, user, "#{user}@mail.com")
  roles = %w[org_admin channel_admin config_admin system_group_admin activation_key_admin image_admin]
  roles.each do |role|
    $api_test.user.add_role(user, role)
  end
  add_context('user', user)
  add_context('password', 'linux')
  log "New user #{user} created"
end
```

Reply:
I see. With the above linked step, that makes sense. Thank you for the clarification, Maxime.